packages/code-studio/src/log/LogInit.ts

```diff
@@ -12,13 +12,15 @@ export const logProxy = new LogProxy();
 export const logHistory = new LogHistory(logProxy);
 
 export default function logInit(): void {
+  Log.setLogLevel(parseInt(process.env.REACT_APP_LOG_LEVEL ?? '', 10));
```

Comment:
When we do a `parseInt`, we should be checking if it's a valid input... for example, `parseInt('', 10)` returns `NaN`, in which case we should fall back to a default value.

EDIT: Nevermind, it already ignores it in `setLogLevel`

Reply:
Ya I chose to ignore it in `setLogLevel` since `typeof NaN === 'number'` so TS won't complain if you pass `NaN`.